### DISCUSSION TITLE
Use parametrize only on one parameter

### DISCUSSION THREAD

**Discussion by mihalt** [pytest-dev/pytest#10629] [General] [answered]
Hi! I would like not to note each constant fixture for each value that I want to put to data parameter like that. As I understand it's not possible by default?
```
@pytest.mark.parametrize('data', [1, 2, 3, 4, 5 ... ])
def test(constant_fixture1, constant_fixture2, constant_fixture3, data):
    ...
```

**Comment by The-Compiler** [comment, marked as answer]
The code you've shown works as-is, assuming that those fixtures are defined. I'm not sure what you mean with "I would like not to note each constant fixture for each value that I want to put to data parameter like that."...

**Reply by mihalt** [reply]
everything is fine, problem was on my side, sorry for disturb
